de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/dialogs/AbstractFileUploadDialogDefaultAppearance.java

```diff
@@ -57,8 +69,28 @@ public String confirmAction() {
     }
 
     @Override
-    public String fileExist() {
-        return iplantErrorStrings.fileExist();
+    public SafeHtml fileExists(String filenames) {
+        return iplantErrorStrings.fileExists(filenames);
+    }
+
+    @Override
+    public String invalidFileName() {
+        return diskResourceMessages.invalidFileName();
+    }
+
+    @Override
+    public String fileNameValidationMsg() {
+        return validationMessages.drNameValidationMsg(
+                validationConstants.newlineToPrint() + validationConstants.tabToPrint()
+                + validationConstants.restrictedDiskResourceNameChars()) + " Offending file(s): ";
+    }
+
+    @Override
+    public String getFileName(String filename) {
+        if (Strings.isNullOrEmpty(filename) || !GXT.isChrome()) {
+            return filename;
+        }
+        return filename.substring(12); //chrome always returns C:\fakepath\filename
```

Comment:
As fas as I know only chrome attaches `C:\fakepath\` to the file name to prevent client from knowing actual path. See https://stackoverflow.com/questions/4851595/how-to-resolve-the-c-fakepath